### PR TITLE
Corrected spinner animation

### PR DESCRIPTION
**Pull request by ranbena**
&nbsp; | Demo
------------ | -------------
❌ Before<br />✅ After | ezgif com-video-to-gif

### The bug
`fa-spinner` was not meant to spin smoothly but in steps.

### And this is important because why? 🙄
I work a lot with Cypress and it's hurting my eyes and crushing my soul! 😱😵😖😭

### The fix
Changed to the dedicated 8 step `fa-pulse` animation.

### Tested?
Nope. But it looks riskless 😅

### CODE REVIEW

File: packages/runner/src/header/header.jsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export default class Header extends Component {
           >
             <input className='url' value={state.url} readOnly onClick={this._openUrl} />
             <span className='loading-container'>
-              ...loading <i className='fa fa-spinner fa-spin fa-pulse'></i>
+              ...loading <i className='fa fa-spinner fa-pulse fa-pulse'></i>
```
Comment (flotwig) [inline packages/runner/src/header/header.jsx]: `fa-pulse fa-pulse`? :thinking:

Comment (ranbena) [reply packages/runner/src/header/header.jsx]: Whoops!

**Comment by flotwig** [comment]
Thanks for the PR!

**Comment by Bkucera** [comment]
@ranbena I see where you're coming from, but if you look at
https://fontawesome.com/v4.7.0/examples/

there are multiple examples of `fa-spinner` with `fa-spin`

I have no preference

**Comment by Bkucera** [comment]
@ranbena Here's how you can add this today. In `support/index.js` do:
```js
addGlobalStyle(`
.fa-spinner.fa-spin {
	animation: fa-spin 1s infinite steps(8)
}
`)

function addGlobalStyle(css) {
  var head, style
  head = window.top.document.getElementsByTagName('head')[0]
  if (!head) {
    return
  }
  style = window.top.document.createElement('style')
  style.type = 'text/css'
  style.innerHTML = css
  head.appendChild(style)
}
```

**Pull request merged**